Apps/Examples/Examples/All Examples/CustomAnchoredSymbolExample.swift

```diff
@@ -0,0 +1,321 @@
+import UIKit
+import MapboxMaps
+import Turf
+
+private enum SymbolTailPosition: Int {
+    case left
+    case right
+    case center
+}
+
+private struct DebugFeature {
+    var coordinate: CLLocationCoordinate2D
+    var highlighted: Bool
+    var sortOrder: Int
+    var tailPosition: SymbolTailPosition
+    var label: String
+    var imageName: String
+}
+
+@objc(CustomAnchoredSymbolExample)
+public class CustomAnchoredSymbolExample: UIViewController, ExampleProtocol {
+    static let symbols = "custom_symbols"
```

Comment:
Could we improve the naming of this constant? Is it an identifier?

Reply:
It also seems unnecessary for this to be static. Letting it be an instance property would clean things up at the point of use.